unskript-ctl/creds_ui.py

```diff
@@ -53,7 +53,10 @@
 
 # This variable is used to hold the Credential directory
 # Where all the creds are saved
-CREDS_DIR = os.environ.get('HOME') + "/.local/share/jupyter/metadata/credential-save/"
+if os.environ.get('CREDS_DIR') != None:
```

Comment:
please fix the pylint here, github is showing something

Reply:
Done